clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -34,7 +32,7 @@ class ElasticsearchIndex[Document: FieldMapper](
     * addition, but in GCS's filesystem adapter it's the only indication that this
     * should be treated as a directory, not a file.
     */
-  lazy val rootDir: String = indexName.replaceAll("_", "-") + "/"
+  lazy val rootDir: String = indexName + "/"
```

Comment:
It's mentioned in the PR description, but for anyone guilty of skipping that whole block (like me sometimes): this change makes Elasticsearch index names consistent with corresponding directory names in cloud storage (the indexes used to be named like `wgs_ubam`, but will now be `wgs-ubam` etc). Merging this PR will trigger a full dev rebuild because of the breaking index-name change, so we should think about any PRs that we want to merge before this one.